wct-core/src/main/resources/defaultH3Profile.cxml

```diff
@@ -34,7 +34,7 @@
  <bean id="simpleOverrides" class="org.springframework.beans.factory.config.PropertyOverrideConfigurer">
   <property name="properties">
    <value>
-metadata.operatorContactUrl=http://www.natlib.govt.nz/
+metadata.operatorContactUrl=ENTER_AN_URL_WITH_YOUR_CONTACT_INFO_HERE_FOR_WEBMASTERS_AFFECTED_BY_YOUR_CRAWL
```

Comment:
Would it be clear to the user that the URL must include `http://`?

Reply:
Will add http(s):// into that string. I was hoping we would add some tooltips to this page at some point to help with info like this